### PR TITLE
NV-3548 - 🐛 Bug Report: Can't embed hyperlink in Slack message

### PR DESCRIPTION
**Pull request by gitstart**
### What change does this PR introduce?
Allows users to embed links in the message.

 ### Why was this change needed?
Closes: https://github.com/novuhq/novu/issues/3548

### Other information

 [Loom Video](https://www.loom.com/share/83cec8292cdb452eaaceb03bb453dc14?sid=33b2553c-a208-43d9-a7aa-790d7af983e3)

---
This code was written and reviewed by GitStart Community. Growing great engineers, one PR at a time.

### CODE REVIEW

**Comment by gitstart** [comment]
@scopsy this PR is ready for review

**Comment by gitstart** [comment]
@scopsy this PR is ready for review

File: apps/web/src/pages/templates/components/TemplateEditorFormProvider.tsx [hunk 1]
```diff
@@ -42,7 +42,7 @@ const makeStep = (channelType: StepTypeEnum, id: string): IFormStep => {
       subject: '',
       type: channelType,
       content: channelType === StepTypeEnum.EMAIL ? defaultEmailBlocks : '',
-      contentType: 'editor',
+      contentType: 'customHtml',
```
Comment (scopsy) [inline apps/web/src/pages/templates/components/TemplateEditorFormProvider.tsx:45]: @gitstart this change will modify all existing channel which is something we don't want. This fix should only apply to the "Chat" channel and not on all available channels 🙏

**Review by scopsy — CHANGES_REQUESTED** [review]

File: apps/web/src/pages/templates/components/TemplateEditorFormProvider.tsx (hunk 1 above)
Comment (gitstart) [reply apps/web/src/pages/templates/components/TemplateEditorFormProvider.tsx]: @scopsy sorry for the delay on this. The latest updates resolves this. Here is how it renders on slack:

image

**Comment by scopsy** [comment]
@gitstart looks better! Is this change backwards. compatible if the user had content on the step previously in the editor type?

**Comment by gitstart** [comment]
@scopsy the change only affects new ones. So no. It's not backwards compatible in that case.

**Comment by gitstart** [comment]
@scopsy How do you want to proceed with this?

**Comment by scopsy** [comment]
@gitstart we would need to validate that old workflows with chat content, are still rendered after this change in the `customHtml` editor. I believe this should work, but will need to validate this.

**Pull request closed without merging**